frontend/src/api/ApiCaller.tsx

```diff
@@ -225,6 +231,14 @@ export class BackendAPICaller {
         return result.content
     }
 
+    static async setArmPosition(robotId: string, armPosition: string): Promise<void> {
+        const path: string = `robots/${robotId}/SetArmPosition/${armPosition}`
+        const result = await BackendAPICaller.PUT(path).catch((e) => {
```

Comment:
No need to store return value from the BackendAPICaller, as it is empty
```suggestion
        await BackendAPICaller.PUT(path).catch((e) => {
```

Reply:
done